src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -238,3 +240,47 @@ def github_comment_workflow(pullrequest_id: int, project_id: int):
 
         metrics.incr("github_pr_comment.api_error")
         raise e
+
+
+@instrumented_task(name="sentry.tasks.integrations.github_comment_reactions")
+def github_comment_reactions():
+    logger.info("github.pr_comment.reactions_task")
+
+    comments = PullRequestComment.objects.filter(
+        created_at__gte=datetime.now(tz=timezone.utc) - timedelta(days=30)
+    )
+
+    for comment in RangeQuerySetWrapper(comments):
+        pr = comment.pull_request
```

Comment:
You should have `select_related('pull_request')` on your comment query to avoid an N+1.